Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/IMixedRealityInputTypeDatum.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
+{
+    public interface IMixedRealityInputTypeDatum<TReading>
```

Comment:
need doc comments

Reply:
agreed

Reply:
I think this interface is somewhat redundant. Input Events and the capabilities will determine what is availible and when.